pandas/tools/merge.py

```diff
@@ -1411,6 +1411,10 @@ def __init__(self, objs, axis=0, join='outer', join_axes=None,
             sample = objs[0]
         self.objs = objs
 
+        # Check for string axis parameter
+        if isinstance(axis, str):
+            axis = objs[0]._get_axis_number(axis)
```

Comment:
@jreback Does there already exist a utility function similar to `_get_axis_number` that does not depend on the actual object?